src/python/version.txt

```diff
@@ -1 +1 @@
-1.2.14
\ No newline at end of file
+1.2.15
```

Comment:
The version gets auto incremented https://github.com/vmware-labs/client-library-for-chaos-mesh/blob/db0e9ccfcc43d4aad536e6e906193a6d4fc56a9d/src/python/setup.py#L14